src/lib/stellar-wallets-kit.ts

```diff
@@ -172,20 +185,33 @@ export class StellarWalletsKit {
 
     switch (this.selectedWallet) {
       case WalletType.XBULL:
+        if ("blob" in params) {
```

Comment:
I'm not sure what you prefer for wallets that don't support this yet, but I think throwing an error is appropriate for now.

Reply:
Yes I agree for now it's ok to just throw an error